src/Components/ResourceList/Card/ResourceCardEditableName.scss

```diff
@@ -19,8 +19,8 @@
   position: relative;
   display: flex;
   flex-wrap: nowrap;
-  align-items: center;
```

Comment:
Dont need it in center, rather, at the start of the flex.

Reply:
one thing to be aware of when modifying CSS is where this class is used. I did a quick poke around the UI (in the chrome console, using `$('.cf-resource-editable-name)` will show you if there are any elements that match that class selector) and found a few places like in paginated tasks and paginated tokens where this class is used. Will justifying the content to `flex-start` mess things up there?

I did notice that changing `align-items: center` changed the vertical placement of text. Do you know what happens when you keep `align-items: center` and add `justify-content: flex-start`?